examples/ct/ct_handlers.go

```diff
@@ -193,7 +190,7 @@ func enforceMethod(w http.ResponseWriter, r *http.Request, method string) bool {
 
 	// For GET requests all params come as form encoded so we might as well parse them now.
 	// POSTs will decode the raw request body as JSON later.
-	if r.Method == httpMethodGet {
+	if r.Method == http.MethodGet {
```

Comment:
I think these were added in Go 1.6? I assume we're ok with not supporting earlier versions but not certain.